packages/main/src/CheckBox.ts

```diff
@@ -321,7 +321,7 @@ class CheckBox extends UI5Element implements IFormElement {
 		const formSupport = getFeature<typeof FormSupport>("FormSupport");
 		if (formSupport) {
 			formSupport.syncNativeHiddenInput(this, (element: IFormElement, nativeInput: HTMLInputElement) => {
-				nativeInput.disabled = element.disabled || !element.checked;
+				nativeInput.disabled = element.disabled || (!element.required && !element.checked);
```

Comment:
I'm curious if we need that line at all. Do you know what would happen if we remove the line?

Reply:
My bad. disabled property of the native input should be linked with the disabled property of the ui5-checkbox so: `nativeInput.disabled = element.disabled || false` seems to be more correct

Reply:
Hi @nnaydenow,

Thank you very much for the feedback.
I made the necessary changes.

Reply:
Sorry again, but we I saw that checbkox should also provide checked information when validating an required attribute. Let's also add `nativeInput.checked = !!element.checked`